iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Bus.java

```diff
@@ -17,7 +17,7 @@
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public interface Bus extends Identifiable<Bus> {
+public interface Bus extends Identifiable<Bus>, VoltageLimitsHolder {
```

Comment:
I saw that the branch doesn't inherit from `FlowsLimitsHolder`. Do I miss something?

To be checked: how calculated bus manage limits.

Reply:
What I made was: when setting a calculated bus (NB to BB), it is actually the busbarsections underneath which are set (an error is thrown if there is none). For a merged bus (BusBreaker to Bus), it is actually the BB buses underneath which are set.

Reply:
Changed a bit the mecanism in the new commits

Reply:
I've thought a bit about your proposition and I don't think it is relevant to make `Branch` inherit from `FlowsLimitHolder`: indeed, in any cases, it hasn't any common methods with flow limit holder without sides so if we needed to iterate on `FlowsLimitHolder`, we would at the end cast it as `Branch` to use it so it is a bit pointless...